src/Quartz.OpenTracing/QuartzDiagnosticOptions.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+
+namespace Quartz.OpenTracing
+{
+    public class QuartzDiagnosticOptions
```

Comment:
I think OpenTelemetry share the same concepts here. Should these options be unified to be quite the same between the integrations, thoughts?

Reply:
Yes, it looks similar and indeed OpenTracing and OpenTelemetry have the same concept, but there're difference between implementations. 

- Looks like in OpenTelemetry implementation we try to subscribe to Veto event also but I think for tracing we interest only for Execute event cuz we need tracing some long operations not just logging some statements
- In OpenTracing impl we have only event name and payload from diagnostic and then we manually create span and fill all tags ourself therefore we've `ComponentName` property and `OperationNameResolver` func in the options and users can override these as they want. At the same time as I can see OpenTelemetry gets already filled Activity and can only add some tags, it can't change OperationName cuz it's already filled from DiagnosticListener.
- If we merge these options we'll must move it to `Quartz` project or will create some new project to share options. I don't think so it's good idea to move it to `Quartz`, it's nothing connecting to main project it's implementation details of tracing.
And then we have to support it between impls and I expect that after OpenTelemetry is released it'll be main approach for distributing tracing and it can develop faster.

But I'm not familiar with OpenTelemetry implementation so well and mb I don't know something.

Reply:
I also can add example code in `Quartz.Examples.AspNetCore` project but it looks like so overloaded already.

And I suggest to create another issue for `docs`.